refinery_migrations/Cargo.toml

```diff
@@ -24,7 +24,7 @@ mysql = {version = "16.0", optional = true}
 chrono = "0.4"
 walkdir = "2.2"
 serde = { version = "1.0", features = ["derive"] }
-cfg-if = "0.1"
+cfg-if = "0.1.10"
```

Comment:
I think you want `"=0.1.10"` if you want to lock that version.

Reply:
yeah, missed that thanks!